packages/node_modules/@webex/internal-plugin-calendar/test/unit/spec/calendar.js

```diff
@@ -0,0 +1,84 @@
+/*!
+ * Copyright (c) 2015-2019 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import {assert} from '@webex/test-helper-chai';
+import Calendar from '@webex/internal-plugin-calendar';
+import MockWebex from '@webex/test-helper-mock-webex';
+import btoa from 'btoa';
+import sinon from 'sinon';
+
+describe('internal-plugin-calendar', () => {
+  describe('Calendar Apis', () => {
+    const webex = new MockWebex({
+      children: {
+        calendar: Calendar
+      }
+    });
+
+    beforeEach(async () => {
+      webex.canAuthorize = true;
+
+      webex.request = sinon.stub().returns(
+        Promise.resolve({
+          body: {
+            items: [
+              {id: 'calendar1', encryptedNotes: 'notes1', encryptedParticipants: ['participant1']},
+              {id: 'calendar2', encryptedNotes: 'notes2', encryptedParticipants: ['participant2']}
+            ]
+          }
+        })
+      );
+    });
+
+    describe('Public Api Contract', () => {
+      describe('#getNotes()', () => {
+        it('should fetch the meeting notes', async () => {
+          const id = 'meetingId123';
+
+          webex.request = sinon.stub().returns(
+            Promise.resolve({
+              body: {
+                encryptedNotes: 'notes1'
+              }
+            })
+          );
+
+          await webex.internal.calendar.getNotes(id)
```

Comment:
Lets not mix async and promises:

```suggestion
          const res = await webex.internal.calendar.getNotes(id);
          assert.equal(res.body.encryptedNotes, 'notes1');
```